src/data-catalog/data-browser-utils.ts

```diff
@@ -43,7 +43,7 @@ export const getDatasetReleasePoliciesDisplayInformation = (
     [Utils.DEFAULT, () => ({ label: dataUsePermission! })]
   );
 
-export const makeDatasetReleasePolicyDisplayInformation = (dataUsePermission: string): ReactElement => {
+export const makeDatasetReleasePolicyDisplayInformation = (dataUsePermission: string | undefined): ReactElement => {
```

Comment:
We seem to use both `?` and `| undefined`. Is there a reason to use `| undefined` here? The function this calls, `getDatasetReleasePoliciesDisplayInformation`, uses `?`.